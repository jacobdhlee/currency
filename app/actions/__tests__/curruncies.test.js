import {
  SWAP_CURRENCY,
  swapCurrecy,
  changeCurrencyAmount,
  CHANGE_CURRENCY_AMOUNT,
} from '../currencies';

it('create SWAP_CURRENCY action', () => {
  const expected = {
    type: SWAP_CURRENCY,
  };
  const result = swapCurrecy();
  expect(result).toEqual(expected);
});

it('create CHANGE_CURRENCY_AMOUNT action', () => {
  const expected = {
    type: CHANGE_CURRENCY_AMOUNT,
    amount: 5000,
  };
  const result = changeCurrencyAmount(5000);
  expect(result).toEqual(expected);
});

it('create SWAP_CURRENCY snapshot', () => {
  expect(swapCurrecy()).toMatchSnapshot();
});

it('create CHANGE_CURRENCY_AMOUNT snapshot', () => {
  expect(changeCurrencyAmount(5000)).toMatchSnapshot();
});
