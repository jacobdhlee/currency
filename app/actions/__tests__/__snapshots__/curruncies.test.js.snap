// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create CHANGE_CURRENCY_AMOUNT snapshot 1`] = `
Object {
  "amount": 5000,
  "type": "CHANGE_CURRENCY_AMOUNT",
}
`;

exports[`create SWAP_CURRENCY snapshot 1`] = `
Object {
  "type": "SWAP_CURRENCY",
}
`;
